pages/common/logstash.md

```diff
@@ -0,0 +1,12 @@
+# logstash
+
+> Migrate data from one source into another.
```

Comment:
I want to keep the part that we can transform the information too. Any way you can think of including that part ?